tests/unit/test_prefixer.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import re
+
+import pytest
+
+import test_utils.prefixer
+
+
+class FakeDateTime(object):
+    """Fake datetime class since pytest can't monkeypatch attributes of
+    built-in/extension type.
+    """
```

Comment:
Would using `freezegun` represent a viable alternative?

True, it would be a new dependency, but it's widely used and maintained, and supports a wide range of features.

(the downside is that the more recent versions require Python 3.5+, but it could still be an option when Python 2.7 support is finally dropped in all libraries)

Reply:
Yeah, freezegun would have been the more robust choice. Indeed, I didn't want to introduce a new dependency just for this.